src/components/Images/Images.js

```diff
@@ -1,31 +1,17 @@
-import React from 'react';
-import { Link } from 'react-router';
+import React, { PropTypes as T } from 'react';
 
-const Images = ({ photos }) => (
+import Image from './Image';
+
+export default function Images({
```

Comment:
I changed this from `const myFunc = () => ...` to `export default function...` because of our discussion on Slack but really I don't have a preference either way. Let me know what you want to use and we can stick to that. :)